cm_service_restart.py

```diff
@@ -36,6 +36,16 @@ def select_from_items(item_type, items):
         print 'Bad input received...'
         return select_from_items(item_type, items)
 
+# Rolling restart method
+def rolling_restart(service_RR):
+    print service_RR
+    # batch_size =
+    # fail_count_threshold =
+    # stale_config_only = select_from_items('Stale config (true/false') # Needs to be a user input
```

Comment:
looks like there is more to do here, can you walk me through what youre thinking? i guess these are all args that need to be passed to the rolling restart?